opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json

```diff
@@ -148,14 +157,60 @@
         }
       },
       {
-        "key": "danger_signs_present_toaster",
+        "key": "danger_signs_present_child_toaster",
         "openmrs_entity_parent": "",
         "openmrs_entity": "concept",
         "openmrs_entity_id": "",
         "type": "toaster_notes",
-        "text": "Danger sign! \nRefer immediately to health facility.",
-        "text_color": "#CF0800",
-        "toaster_type": "problem",
+        "text": "We've identified that the baby has danger signs. Do not worry, because these conditions will be managed when you get to a health facility?",
+        "toaster_info_text": "We've identified that the baby has danger signs. Do not worry, because these conditions will be managed when you get to a health facility?",
+        "toaster_type": "warning",
+        "relevance": {
+          "step1:danger_signs_present_child": {
+            "ex-checkbox": [
+              {
+                "or": [
+                  "chk_fever",
+                  "chk_excessive_crying",
+                  "chk_fast_breathing",
+                  "chk_difficult_breathing",
+                  "chk_unable_to_suck",
+                  "chk_umbilical_discharge",
+                  "chk_convulsions",
+                  "chk_skin_rashes",
+                  "chk_pale",
+                  "chk_severe_abdominal_pain",
+                  "chk_lethargy",
+                  "chk_urination"
+                ]
+              }
+            ]
+          }
+        }
+      },
+      {
+        "key": "neonatal_visited_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
We are phasing out the use of openmrs uuids here, please add the same value as the key